docs/getting-started/smr-disk.md

```diff
@@ -250,14 +250,16 @@ also be checked using the [*libzbc* conformance test suite](/tests/zbc-tests).
 
 ### Verifying The Disk
 
-Assuming that a compatible HBA is being used, after connecting the drive and
-eventually rebooting the system (most SAS HBAs have plug-and-play features in
-which case rebooting the system after connecting or disconnecting a disk is
-not necessary), verifying the disk identification and checking the disk
-parameters and zone configuration can be done in the exact same manner as with
-Serial ATA disks as discussed above.
-
-In these examples, `/dev/sdc` is a SAS disk connected to a SAS HBA and
+If you are using a compatible HBA and you have connected the drive and rebooted
+the system, verifying the disk identification and checking the disk parameters
+and zone configuration can be done in the exact same manner as with Serial ATA
+disks as discussed above.
+
+(Remember that most SAS HBAs have plug-and-play features that make it
+unnecessary to reboot the system after connecting or disconnecting a disk to
+it.) 
+
+In these examples, `/dev/sdc` is an SAS disk connected to an SAS HBA and
```

Comment:
For my education, why is "an" needed here ? When the next letter/sound is consonant, shouldn't "a" be used ? Not entirely sure about the English rules here.

Reply:
See the comment below. (GitHub (which I also desperately hate) doesn't nest replies properly when they're made from external email clients.)